src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/sealedclaim/SealedClaimFormGenerator.java

```diff
@@ -63,14 +68,19 @@ private String getFileName(DocmosisTemplates docmosisTemplate, CaseData caseData
     public SealedClaimForm getTemplateData(CaseData caseData) {
         Optional<SolicitorReferences> solicitorReferences = ofNullable(caseData.getSolicitorReferences());
         MultiPartyScenario multiPartyScenario = getMultiPartyScenario(caseData);
-
+        List<LocationRefData> courtLocations = (locationRefDataService
+            .getCourtLocationsByEpimmsId(
+                CallbackParams.Params.BEARER_TOKEN.toString(),
+                caseData.getCourtLocation().getCaseLocation().getBaseLocation()));
         SealedClaimForm.SealedClaimFormBuilder sealedClaimFormBuilder = SealedClaimForm.builder()
             .applicants(getApplicants(caseData, multiPartyScenario))
             .respondents(getRespondents(caseData, multiPartyScenario))
             .claimValue(caseData.getClaimValue().formData())
             .statementOfTruth(caseData.getApplicantSolicitor1ClaimStatementOfTruth())
             .claimDetails(caseData.getDetailsOfClaim())
-            .hearingCourtLocation(caseData.getCourtLocation().getApplicantPreferredCourt())
+            .hearingCourtLocation(courtLocations.isEmpty() ? null : courtLocations.stream()
+                .filter(id -> id.getCourtTypeId().equals("10"))
```

Comment:
what is this magic number `10` here, can we please create a constant with descriptive name so it's obvious in future

Reply:
10 is the court type id for CIVIL, I have created a constant.